src/api/EventsAPI.ts

```diff
@@ -19,30 +16,101 @@ export type EventObject = {
   pointValue: number;
   requiresStaff: boolean;
   staffPointBonus: number;
+  facebookUrl?: string;
 };
 
-const handleErrors = (response: Response): Promise<EventsResponse> => {
+export type EventsArray = EventObject[];
+
+export type EventsResponse = {
+  error: unknown;
```

Comment:
why is this type `unknown`? what shape does it actually end up taking (@StormFireFox1 is there a standardized error type we can expect to get from portal)

Reply:
from what I can tell, it's impossible to throw an error in the first place but idk?
 
![Screen Shot 2022-03-24 at 12 44 28 AM](https://user-images.githubusercontent.com/33165426/159867001-5527d953-4489-4d24-9890-05d04dc1903e.jpg)
e

Reply:
We will be publishing our error type during the workspace refactor on portal backend, so you can use that here once that's done. It is possible for errors to be thrown here, e.g. if an invalid query parameter is supplied, or if there's some internal database error, so we can't rule that out entirely (though realistically this route shouldn't throw many errors at all)